tutorials/transfer_learning/transfer_learning.jl

```diff
@@ -65,7 +67,7 @@ opt = ADAM()
 loss(x,y) = Flux.Losses.logitcrossentropy(model(x), y)
 
 # Now to train
-# As discussed earlier, we don’t need to pass all the parameters to our training loop. Only the ones we need to
+# As discussed earlier, we don't need to pass all the parameters to our training loop. Only the ones we need to
 # fine-tune. Note that we could have picked and chosen the layers we want to train individually as well, but this
 # is sufficient for our use as of now.
 
```

Comment:
I can't comment below, probably this should remove `@epochs` in favour of `for epoch in 1:2`.

A bigger change would be to move on from implicit parameters, `params` etc, but perhaps that can be the next PR.

Reply:
I made the basic fix. I don't think it's worth making the explicit parameter changes until Metalhead gets an update as the current Metalhead stable version's compat prevents Flux from updating past 0.13.4 because of the maximum versions for Functors and MLUtils.